### PR TITLE
Added 3D visualization of composite model interfaces

### PR DESCRIPTION
**Pull request by robbr48**
They are also highlighted when a connection is selected.

### CODE REVIEW

**Comment by adeas31** [comment]
Can you rebase it?

File: OMEdit/OMEditGUI/Modeling/ModelWidgetContainer.cpp [hunk 1]
```diff
@@ -3666,10 +3666,185 @@ bool ModelWidget::writeVisualXMLFile(QString fileName, bool canWriteVisualXMLFil
...
+        visualFile << "      <exp>0</exp>\n";
+        visualFile << "      <exp>0</exp>\n";
+        visualFile << "      <exp>1</exp>\n";
+        visualFile << "    </lengthDir>\n";
+        visualFile << "    <widthDir>\n";
+        visualFile << "      <exp>1</exp>\n";
+        visualFile << "      <exp>0</exp>\n";
+        visualFile << "      <exp>0</exp>\n";
+        visualFile << "    </widthDir>\n";
+        visualFile << "    <length><exp>0.5.0</exp></length>\n";
```
Comment (adeas31) [inline OMEdit/OMEditGUI/Modeling/ModelWidgetContainer.cpp]: `0.5.0`? Looks like a typo.

Comment (robbr48) [reply OMEdit/OMEditGUI/Modeling/ModelWidgetContainer.cpp]: Ops, I fixed it now.

**Comment by robbr48** [comment]
I squashed all commits related to the fix. I wonder if this PR cannot be rebased because my last PR was from the default branch and the branch for this PR contains some already pushed commits? Any ideas how to fix it?

**Pull request closed without merging**